database/sqlite3/sqlite3.go

```diff
@@ -97,6 +97,11 @@ func (m *Sqlite) Open(url string) (database.Driver, error) {
 		return nil, err
 	}
 	dbfile := strings.Replace(migrate.FilterCustomQuery(purl).String(), "sqlite3://", "", 1)
+	dbfile, err = nurl.QueryUnescape(dbfile)
```

Comment:
The whole connection string cannot be unescaped as [the options need to be escaped](https://github.com/mattn/go-sqlite3#connection-string).
I think we'll need to reconstruct the connection string w/ the host & path unescaped unless you can think of a cleaner way...

[Example](https://play.golang.org/p/4h83CEXdyep) of what the parsed connection string looks like